packages/app/src/components/VisualizationOptions/Options/RangeAxisSteps.js

```diff
@@ -2,11 +2,10 @@ import React from 'react'
 
 import i18n from '@dhis2/d2-i18n'
 
-import TextBaseOption from './TextBaseOption'
+import PositiveNumberBaseType from './PositiveNumberBaseType'
 
 export const RangeAxisSteps = () => (
-    <TextBaseOption
-        type="number"
+    <PositiveNumberBaseType
         width="96px"
         helpText={i18n.t(
             'The number of axis steps between the min and max values'
```

Comment:
Should this help text be changed? Discussion: https://dhis2.slack.com/archives/G3TL0J145/p1595923304005000